synapse/config/ratelimiting.py

```diff
@@ -174,6 +185,14 @@ def generate_config_section(self, **kwargs):
         #rc_3pid_validation:
         #  per_second: 0.003
         #  burst_count: 5
+        #
+        #rc_invites:
+        #  per_room:
+        #    per_second: 0.3
+        #    burst_count: 10
+        #  per_user:
+        #    per_second: 0.03
```

Comment:
Did you mean for these two `per_second`s to be different?

Reply:
Yes, so we want a relatively high `per_room` ratelimit to allow room upgrades, but we can limit the number of invites a user receives a lot (one every 5 minutes for `0.003`)